apm-protocol/apm-network/src/main/java/org/apache/skywalking/apm/network/trace/component/command/ConfigurationDiscoveryCommand.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.skywalking.apm.network.trace.component.command;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.skywalking.apm.network.common.v3.Command;
+import org.apache.skywalking.apm.network.common.v3.KeyStringValuePair;
+
+public class ConfigurationDiscoveryCommand extends BaseCommand implements Serializable, Deserializable<ConfigurationDiscoveryCommand> {
+    public static final Deserializable<ConfigurationDiscoveryCommand> DESERIALIZER = new ConfigurationDiscoveryCommand(
+        "", "", "", new ArrayList<>());
+    public static final String NAME = ConfigurationDiscoveryCommand.class.getSimpleName();
+
+    public static final String UUID_CONST_NAME = "UUID";
+    public static final String SERIAL_NUMBER_CONST_NAME = "SerialNumber";
+    public static final String SERVICE_CONST_NAME = "SERVICE";
+
+    /*
+     * If config is unchanged, then could response the same uuid, and config is not required.
+     */
+    private String uuid;
+    /*
+     * Current service name.
+     */
+    private String service;
```

Comment:
Why need this?

Reply:
I want to check whether the service and agent service of the returned object are the same on the agent side.

Reply:
How could it not same?

Reply:
Just for verification, it has been deleted.